### ISSUE TITLE
Live reload not working on Windows

### ISSUE THREAD

**Issue by lucasmolinari** [Keats/gutenberg#2593]
# Bug Report

## Environment

Zola version: 0.19.1
OS: Windows 10 Pro 10.0.19045
Terminal: Windows Terminal with Powershell

## Expected Behavior
`zola serve` should apply file changes to live server.

## Current Behavior
Making changes in files doesn't trigger live reload and doesn't apply changes. Tried changing files from the subdirectories:
- `.md` from `content/`
- `.scss` from `sass/`
- `.html` from `templates/`

`zola serve` outputs: 
```console
Listening for changes in \\?\C:\Users\PC\codes\myapp\{config.toml,content,sass,static,templates}
Press Ctrl+C to stop

Web server is available at http://127.0.0.1:1111 (bound to 127.0.0.1:1111)
```

## Step to reproduce
Run `zola serve` on Powershell, change a file and see if the changes are applied to the live server.

**Comment by Keats** [comment]
Can you try the `next` branch? It should be fixed

**Comment by lucasmolinari** [comment]
Yes! fixed on the `next` branch.

**Comment by Keats** [comment]
Ok cool, I'll release 0.19.2 this weekend then to unblock the Windows users

**Comment by JVimes** [comment]
Should 0.19.2 be available via winget? I still get 0.19.1.
